src/packages/client/src/runtime/getPrismaClient.ts

```diff
@@ -307,6 +315,12 @@ export function getPrismaClient(config: GetPrismaClientOptions): any {
       }
       return this.disconnectionPromise
     }
+
+    /**
+     * Re-export of 'sql-template-tag'
+     */
+    sql = sqlTemplateTagExport
```

Comment:
You need to add this in `TSClient.ts` in the folder `generation` so the typing follows.

Reply:
Thanks, I think I've added the correct exports, please let me know. :-)